docs/source/models.rst

```diff
@@ -499,3 +499,32 @@ Spark cluster and used to score the model.
 
     pyfunc_udf = mlflow.pyfunc.spark_udf(<path-to-model>)
     df = spark_df.withColumn("prediction", pyfunc_udf(<features>))
```

Comment:
Shouldn't we talk about what `result_type` is assumed if you don't pass one? We should say something about that as a transition into the next paragraph. For example, by default, we return the first numeric column as a double or whatever it is.